pages/1.11/installing/ent/custom/opt-out/index.md

```diff
@@ -1,23 +0,0 @@
----
```

Comment:
Opt out docs should not be deleted. Replace and make sure to reconcile open source and enterprise content.

Reply:
@judithpatudith I've asked @masonse @bernadinm @justinrlee to provide inputs here. I've add opt-out docs based on their feedback.